internal/commands/run_test.go

```diff
@@ -221,14 +221,14 @@ func TestRunContainerPull(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	testutil.TempChdirPopulatedWith(t, "testdata/docker-no-pull")
+	testutil.TempChdirPopulatedWith(t, "testdata/image-pulling-behavior")
 
 	// Create os.Stderr writer that duplicates it's output to buf
 	buf := bytes.NewBufferString("")
 	writer := io.MultiWriter(os.Stderr, buf)
 
 	command := commands.NewRootCmd()
-	command.SetArgs([]string{"run", "-v", "-o simple", "--container-pull"})
+	command.SetArgs([]string{"run", "-v", "-o simple"})
```

Comment:
Should we put `--container-lazy-pull` to all other `run` invocations in test so the tests are graceful on the Docker Hub limits?

Reply:
It's already working this way, see `EagerPull` field in `ContainerOptions`.

Reply:
I believe `EagerPull` is set to `true` in tests because `containerLazyPull` is `false` by default and `EagerPull` is `!containerLazyPull`.

Reply:
Oh indeed, I was thinking of all other tests that don't use the `run` command.

Reply:
Fixed in https://github.com/cirruslabs/cirrus-cli/pull/295/commits/e578304884f6dd2d1b27aa0586a6f8d0b038d6df.